os-lifecycle-policy.md

```diff
@@ -28,12 +26,8 @@ Operating system versions are added and go out of support on a regular basis. We
 * The support documents listed above
 * [Monthly Update Announcements](https://github.com/dotnet/announcements/labels/Monthly-Update)
 
-When a new operating system version is made available, we will typically support it on a subset of supported .NET Core versions, potentially only with the latest .NET Core version.
-
 ## Exceptions to the supported platform policy
 
-Occasionally, we may not adhere to a strict interpretation of the policy for supported OS platforms. Instead, we may choose to offer support on an OS platform that is out of support. Or we may choose to exclude a supported platform for specific reasons.
+Occasionally, we may choose to offer support on an OS platform that is out of support. Or we may choose to exclude a supported platform for specific reasons.
 
 One example is Windows 7 SP1. Windows 7 SP1 is out of support, but enterprise customers can buy [Extended Security Updates](https://docs.microsoft.com/troubleshoot/windows-client/windows-7-eos-faq/windows-7-extended-security-updates-faq), and many do. Given the widespread use of Windows 7 in the .NET ecosystem we chose to include Windows 7 SP1 (with Extended Security Updates installed) as a supported platform for .NET 5.0. Another example is Windows Server 2012. Windows Server 2012 is currently in support but not included in the supported platform list for .NET 5.0 because a newer version is available in the Server 2012 family - Server 2012 R2.
-
-**_When there is a conflict between our general policy for supported platforms and the specific supported platforms documented for a given release of .NET the supported platform list for the specific version of .NET will supersede the general policy._**
```

Comment:
I don't think we should delete this para - someone will eventually believe there is a mistake somewhere because something we do doesn't match expectations (like the Win7 case). Having a clear statement that the date grid trumps general policy is useful as a clarification.

Reply:
I think the wording is unnecessary. Respectfully, It's also very difficult for me to parse. I also don't think we need wording to account for supporting Win 7 longer than expected, beyond what I kept in the document. This isn't a legal document that has to account for all eventualities. When we make a choice that we expect will not align with expectations, it is incumbent on us to announce (and explain) ahead of time. I don't think the Win7 case, however, required such an announcement. It may be important to people, but it's "no big deal" relative to our policies.